model.save() with custom id does not work

I try this

```js
model.set('id', `myid`)
model.save()
```

but get this error

> ember.debug.js:31321 Error: Assertion Failed: An adapter cannot assign a new id to a record that already has an id. <coach@model:measurement::ember364:myid> had id: myid and you tried to update it with 0DC98426-0DD3-9B38-8F4A-5881C2E4A8E5. This likely happened because your server returned data in response to a find or update that had a different id than the one you sent.

If I work against the standard store, it sends a POST request and the id is in the body, so must be something with our adapter

--- a/addon/adapters/hoodie.js
+++ b/addon/adapters/hoodie.js
@@ -21,6 +21,7 @@ export default DS.Adapter.extend({
 
   createRecord(store, type, snapshot) {
     var props = this.serialize(snapshot);
+    if (snapshot.id) { props.id = snapshot.id }
     return this._next(() => {
       return this._storeForType(type).add(props);
     });

